### PR TITLE
Client side zc

### PR DESCRIPTION
**Pull request by ZhuoChenP**

### CODE REVIEW

File: ezparking_client/src/components/header/index.js [hunk 1]
```diff
@@ -2,13 +2,13 @@ import logo from '../../assert/logo.png';
 
 function PageHeader() {
     return ( 
-    
     <div className='inline-flex mx-auto my-3'>  
         <img className='h-10' src={logo} alt="Logo" />
```
Comment (kevinren1108) [inline ezparking_client/src/components/header/index.js:6]: add a ```<a>``` tag to return home would be nice, be sure to clean the redirect state. Other than this enhancement, everything else looks great!

**Review by kevinren1108 — APPROVED** [review]

**Review by Kecheng-Yu — APPROVED** [review]
nice work and meet my expected

**Comment by ziwentan** [comment]
that is great.

**Review by ziwentan — APPROVED** [review]
that's good.

**Pull request merged**